### PR TITLE
daemon: switch to batch mode when available

### PR DESCRIPTION
**Pull request by pkalever**
### What does this PR achieve? Why do we need it?
latest targetcli daemon approach set interactive mode as default to make it
appear similar to cli approach, hence to use previous batch mode we need
some global settings.

Depends on: targetcli#pull/167

Signed-off-by: Prasanna Kumar Kalever \<prasanna.kalever@redhat.com\>

### CODE REVIEW

File: daemon/gluster-blockd.c [hunk 1]
```diff
@@ -539,9 +539,17 @@ blockNodeSanityCheck(void)
         return ENOMEM;
       }
     } else {
-      if (GB_ASPRINTF(&global_opts, "%s auto_use_daemon=true", tmp) == -1) {
-        GB_FREE(tmp);
-        return ENOMEM;
+      ret = gbRunner("targetcli set global | grep daemon_use_batch_mode");
```
Comment (lxbsz) [inline daemon/gluster-blockd.c:542]: or should s/set/get/ ? though we will get the same result, but it may confusing.

Comment (pkalever) [reply daemon/gluster-blockd.c:542]: @lxbsz 
Maybe get global will have to read the details of each option and display the final output, whereas set global will just display the help. So I thought set will serve the purpose.

Thanks!

Comment (lxbsz) [reply daemon/gluster-blockd.c:542]: Both are okay for me.

**Comment by pkalever** [comment]
@lxbsz updated the tags and merged, thanks!

**Pull request merged**